client/signup/step-wrapper/index.jsx

```diff
@@ -102,28 +151,41 @@ class StepWrapper extends Component {
 
 	render() {
 		const {
+			flowName,
 			stepContent,
 			headerButton,
 			hideFormattedHeader,
 			hideBack,
 			hideSkip,
+			hideNext,
 			isLargeSkipLayout,
 			isWideLayout,
 			skipButtonAlign,
 			align,
 		} = this.props;
 
-		const hasHeaderButtons = headerButton || ( ! hideSkip && skipButtonAlign === 'top' );
+		const hasNavigation = ! hideBack || ( ! hideSkip && skipButtonAlign === 'top' ) || ! hideNext;
 		const classes = classNames( 'step-wrapper', this.props.className, {
 			'is-wide-layout': isWideLayout,
 			'is-large-skip-layout': isLargeSkipLayout,
-			'has-header-buttons': hasHeaderButtons,
+			'has-navigation': hasNavigation,
 		} );
 
 		return (
 			<>
 				<div className={ classes }>
-					{ ! hideBack && this.renderBack() }
+					{ hasNavigation && (
+						<ActionButtons
+							className="step-wrapper__navigation"
+							sticky={ isReskinnedFlow( flowName ) ? null : false }
+						>
+							{ ! hideBack && this.renderBack() }
+							{ ! hideSkip &&
+								skipButtonAlign === 'top' &&
+								this.renderSkip( { borderless: true, forwardIcon: null } ) }
+							{ ! hideNext && this.renderNext() }
```

Comment:
The difference between the skip button and the next button is primary and borderless, try to separate them to reduce the conditional logics